.github/workflows/release.yml

```diff
@@ -78,35 +78,47 @@ jobs:
 
       - name: Create release notes
         if: steps.check_rerun.outputs.rerun == 'false'
-        id: generate-release-notes
```

Comment:
- Avoid using hard-coded strings like "feat:", "break:", "fix:" in the code as they could be prone to errors if the format changes.